README.md

```diff
@@ -426,11 +426,13 @@ There are three types of limits that you can query with:
 Using these in various combinations results in the underlying requests to be made in the smallest size possible and
 the query returns once `record_limit` or `scan_limit` is satisfied. It will attempt to batch whenever possible.
 
-You can thus limit the number of evaluated records, or select a record from which to start, to support pagination:
+You can thus limit the number of evaluated records, or select a record from which to start in order to support pagination.
 
 ```ruby
 Address.record_limit(5).start(address) # Only 5 addresses starting at `address`
 ```
+Where `address` is an instance of the model or a hash {the_model_hash_key: 'value', the_model_range_key: 'value'}:
+Keep in mind that if you are passing a hash to `.start()` you need to explicitly define all required keys in it, including range keys and secondary global indexes, if any, when querying and, range keys, if any, when scanning. Otherwise you'll get an `Aws::DynamoDB::Errors::ValidationException` either for `Exclusive Start Key must have same size as table's key schema` or `The provided starting key is invalid`
```

Comment:
Hmm not sure on this wording since confused by "including ... secondary global indexes".

Maybe something like "You need to explicitly define all required keys, including range keys, depending on table or secondary index signatures otherwise you'll get an .."